src/main/java/appeng/block/misc/BlockInterface.java

```diff
@@ -52,6 +57,26 @@ public BlockInterface()
 	}
 
 	@Override
+	protected IProperty[] getAEStates()
+	{
+		return new IProperty[] { AE_BLOCK_FORWARD, AE_BLOCK_UP, OMNIDIRECTIONAL };
```

Comment:
What I did here is consistent with all existing overrides of getAEStates. So I'd like to leave this as is and change it in a broader change that also allows for extended block state to be more neatly specified.